pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -594,16 +594,16 @@ type (
 	}
 )
 
-// NodeGroupKubeletConfig contains extra config parameters for the kubelet.yaml
-type NodeGroupKubeletConfig map[string]interface{}
+// InlineDocument contains extra config parameters for the kubelet.yaml
```

Comment:
Since you mentioned elsewhere that `InlineDocument` will get used to enable inline IAM policies, I think we should modify this doc comment to reflect that.

Reply:
Ah, yes, I'll generalise the comment, thanks!